test/Toggle/CreateMarketingInformationForToggleTest.php

```diff
@@ -0,0 +1,101 @@
+    {
+        $marketingInformation = new CreateMarketingInformationRequest( 1, "", "", "", "", "",
+            "", "" );
+        $response = $this->createMarketingInformationFroToggle( new MarketableToggleGatewaySpy(),
+            $marketingInformation );
+        $this->assertEquals( [ "1", "", "", "", "", "", "", "" ], $response );
+    }
+
+    /**
+     * @test
+     */
+    public function givenNull_CreateMarketingInformationForToggle_SetsMarketingInformationToEmptyString()
+    {
+        $marketingInformation = new CreateMarketingInformationRequest( 1, null, null, null,
+            null, null, null, null );
+        $response = $this->createMarketingInformationFroToggle( new MarketableToggleGatewaySpy(),
+            $marketingInformation );
+        $this->assertEquals( [ "1", "", "", "", "", "", "", "" ], $response );
+    }
+
+    /**
+     * @test
+     */
+    public function givenMarketingInformation_CreateMarketingInformationForToggle_SetsMarketingInformationToGivenMarketingInformation()
+    {
+        $marketingInformation = new CreateMarketingInformationRequest( 1, "This", "is", "a",
+            "test", "of", "Marketing", "information" );
+        $response = $this->createMarketingInformationFroToggle( new MarketableToggleGatewaySpy(),
+            $marketingInformation );
+        $this->assertEquals( [ "1", "This", "is", "a", "test", "of", "Marketing", "information" ], $response );
+    }
+
+    /**
+     * @test
+     */
+    public function givenDifferentDataType_CreateMarketingInformationForToggle_SetsMarketingInformationToGivenInformationAsString()
+    {
+        $marketingInformation = new CreateMarketingInformationRequest( 1, 1, 2.1, true, 4353536,
+            "", null, -1 );
+        $response = $this->createMarketingInformationFroToggle( new MarketableToggleGatewaySpy(),
+            $marketingInformation );
+        $this->assertEquals( [ "1", "1", "2.1", "1", "4353536", "", "", "-1" ], $response );
+    }
+
+    /**
+     * @test
+     */
+    public function givenNotAllParamaters_CreateMarketingInformationForToggle_SetsEmptyStringForMissinParameters()
+    {
+        $marketingInformation = new CreateMarketingInformationRequest( 1, "hello", "this",
+            "is a test", 123 );
+        $response = $this->createMarketingInformationFroToggle( new MarketableToggleGatewaySpy(),
+            $marketingInformation );
+        $this->assertEquals( [ "1", "hello", "this", "is a test", "123", "", "", "" ], $response );
+    }
+
+    /**
+     * @param MarketableToggleGatewaySpy $gateway
+     * @param CreateMarketingInformationRequest $request
+     * @return \string[]
+     */
+    private function createMarketingInformationFroToggle( $gateway, CreateMarketingInformationRequest $request )
+    {
+        ( new CreateMarketingInformationForToggle( $gateway ) )->execute( $request );
```

Comment:
You pass in the same gateway every time so you don't really need to pass it at all, you can just create it each time.

In fact I would create the CreateMarketingInformationForToggle object and the spy gateway object in the setUp method to simplify things